models/setting.js

```diff
@@ -0,0 +1,14 @@
+module.exports = (sequelize, DataTypes) =>
+  sequelize.define('Setting', {
+    key: {
+      type: DataTypes.TEXT,
```

Comment:
There's varchar and text. From what I understand, text behaves exactly like varchar with no character limit.